eras/alonzo/impl/src/Cardano/Ledger/Alonzo/TxBody.hs

```diff
@@ -137,34 +132,30 @@ import Prelude hiding (lookup)
 data AlonzoTxBodyRaw era = AlonzoTxBodyRaw
   { atbrInputs :: !(Set (TxIn (EraCrypto era))),
     atbrCollateral :: !(Set (TxIn (EraCrypto era))),
-    atbrOutputs :: !(StrictSeq (Core.TxOut era)),
+    atbrOutputs :: !(StrictSeq (TxOut era)),
     atbrCerts :: !(StrictSeq (DCert (EraCrypto era))),
     atbrWdrls :: !(Wdrl (EraCrypto era)),
     atbrTxFee :: !Coin,
     atbrValidityInterval :: !ValidityInterval,
     atbrUpdate :: !(StrictMaybe (Update era)),
     atbrReqSignerHashes :: Set (KeyHash 'Witness (EraCrypto era)),
     atbrMint :: !(MultiAsset (EraCrypto era)),
-    -- The spec makes it clear that the mint field is a
-    -- Cardano.Ledger.Mary.Value.MaryValue, not a Cardano.Ledger.Core.Value.
-    -- Operations on the TxBody in the AlonzoEra depend upon this.
-    -- We now store only the MultiAsset part of a Mary.Value.
```

Comment:
That is because the spec needs to be fixed. The spec does not have a separate type `MultiAsset` like we do in Haskell, that does not contain ADA. We also removed a rule that checks that mint can't contain ADA, precisely because it can't happen by construction. So this comment is wrong, that is why I removed it.

Here I created a ticket that tracks fixining the spec: https://github.com/input-output-hk/cardano-ledger/issues/3207